Backpack/src/main/java/net/skyscanner/backpack/chip/BpkOutlineChip.kt

```diff
@@ -0,0 +1,57 @@
+package net.skyscanner.backpack.chip
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import android.graphics.drawable.GradientDrawable
+import android.util.AttributeSet
+import android.view.Gravity
+import androidx.annotation.ColorInt
+import androidx.annotation.Dimension
+import androidx.core.content.ContextCompat
+import net.skyscanner.backpack.R
+import net.skyscanner.backpack.text.BpkText
+import net.skyscanner.backpack.util.createContextThemeWrapper
+
+class BpkOutlineChip @JvmOverloads constructor(
+  context: Context,
+  attrs: AttributeSet? = null,
+  defStyleAttr: Int = 0
+) : BpkText(createContextThemeWrapper(context, attrs, R.attr.bpkOutlineChipStyle), attrs, defStyleAttr) {
+
+  private val verticalPadding = context.resources.getDimensionPixelSize(R.dimen.bpkSpacingSm)
+  private val horizontalPadding = context.resources.getDimensionPixelSize(R.dimen.bpkSpacingBase)
+
+  init {
+    gravity = Gravity.CENTER_VERTICAL
```

Comment:
I think we should abstract this away to keep this and the normal chip in sync. Can we just extend `BpkChip` here and only change the background and the default background colour? The rest should work fine as it is